components/forms/TextInput/TextInput.test.js

```diff
@@ -87,27 +88,27 @@ describe("Verfify character count restrictions", () => {
 
   it("does not display any message when not enough characters have been typed in", () => {
     const textInput = screen.getByRole("textbox");
-    userEvent.type(textInput, "This is 21 characters");
+    fireEvent.change(textInput, { target: { value: "This is 21 characters" } });
```

Comment:
See comment on switching from userEvent to fireEvent

Reply:
See earlier response